.mergify.yml

```diff
@@ -0,0 +1,20 @@
+queue_rules:
```

Comment:
@filipdjokic do we want to keep this one?

Reply:
Not really, I think it doesn't make any difference.